common/socket/src/main/java/io/helidon/common/socket/SocketContext.java

```diff
@@ -56,6 +58,13 @@ public interface SocketContext {
      */
     String childSocketId();
 
+    /**
+     * The {@link io.helidon.common.configurable.RequestedUriDiscoveryContext} for the socket.
+     *
+     * @return the discovery context set up for the socket
+     */
+    RequestedUriDiscoveryContext requestedUriDiscoveryContext();
```

Comment:
As written above, this should not be part of SocketContext

Reply:
This is specific to HTTP, that is why it should be on webserver (also only server side related, useless for client)

Reply:
Fixed